managed-databases/postgresql-and-mysql/how-to/upgrade-version.mdx

```diff
@@ -0,0 +1,54 @@
+content:
+  h1: How to upgrade your Database Instance engine version
+  paragraph: This page explains how to upgrade your Database Instance engine to a newer version
+tags: managed-database postgresql mysql engine version database engine
+dates:
+  validation: 2023-02-27
+  posted: 2023-02-27
+categories:
+  - managed-databases
+  - postgresql-and-mysql
+---
+ 
+If your Database Instance uses an engine version versions that is in end of life and is no longer maintaned, we recommend that you upgrade your engine version.
+ 
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/how-to/create-a-database) running on 
+</Message>
+
+1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+
+    <Lightbox src="scaleway-rdb-engine-version-upgrade.webp" alt="List of Database Instances in the Scaleway console. A warning sign displays under the name of the Database Instance that has an engine version in end of life. Next to the warning, an upgrade hyperlink displays" />
+2. Click **Upgrade** next to the Database Instance you wish to upgrade the database engine for. A pop-up appears.
+    <Message type="tip">
+      Alternatively you can click the database name or <Icon name="more" /> > **More info**, then click **Upgrade** under **Database engine** in the Database Instances information page.
+    </Message>
+
+    <Message type="note">
+      The **Upgrade** option is only available for Database Instances using engine versions in end of life.
+    </Message>
+
+    <Lightbox src="scaleway-engine-version-upgrade-pop-up.webp" alt="" />
+3. Make sure that you have read and agree with the conditions of the upgrade. 
+    <Message type="note">
+        You can currently can only upgrade to the next available version. This means that if you have a Database Instance running on PostgreSQL 10, you will be ugraded to PostgreSQL 11. If you want to upgrade to the most recent version of an engine, you must create a new Database Instance and [migrate your databases](https://www.scaleway.com/en/docs/tutorials/migrate-databases-instance/).
+    </Message>
+4. Click **Upgrade version**. A new Database Instance is created.
+    <Message type="important">
+      When you upgrade a version, a clone of your current Database Instance with an up-to-date engine version is created. Your current Database Instance will remain available with its original endpoints, and will still be billed.<br /><br />The new, upgraded Database Instance will have new endpoints. To use it, replace the old endpoints with the new across all your applications. The upgrade does impact your original Database Instance. Even after manually updating your endpoints, you can roll back to the old ones at any time.
+    </Message> 
+
```

Comment:
We can add a note about Engine extensions. On PostgreSQL extension are handled and upgraded during the upgrade process